### PR TITLE
[11.0] migrate web_widget_datepicker_options dev yani

### PR DESCRIPTION
**Pull request by yaniaular**
# Example with: 

``` python
options="{'datepicker': {'daysOfWeekDisabled': [0, 6]} }"
```

https://youtu.be/E-kld1xxiyA

### CODE REVIEW

**Comment by yaniaular** [comment]
@oscarolar can you review and merge please?

File: web_widget_datepicker_options/static/src/js/datepicker.js [hunk 1]
```diff
@@ -16,38 +16,28 @@
     You should have received a copy of the GNU Affero General Public License
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
-/*global openerp, _, $ */
-
-openerp.web_widget_datepicker_options = function (instance) {
 
+odoo.define('inteco.datepicker', function(require) {
```
Comment (oscarolar) [inline web_widget_datepicker_options/static/src/js/datepicker.js]: A good rule of thumb is to use `modulename.jsinstancename` in this case would be: `web_widget_datepicker_options.datepicker`

**Review by oscarolar — CHANGES_REQUESTED** [review]

File: web_widget_datepicker_options/static/src/js/datepicker.js (hunk 1 above)
Comment (yaniaular) [reply web_widget_datepicker_options/static/src/js/datepicker.js]: Yes. it's done

**Review by oscarolar — APPROVED** [review]

**Comment by luisg123v** [comment]
@moylop260  or @oscarolar 
Could you merge this PR, please?

**Pull request merged**